tuist/tuist
# Fix SwiftPackageManager copy rule parsing

Resolves https://github.com/tuist/tuist/issues/4724

### Short description 📝

This PR tries to resolves the issue of tuist not respecting the `.copy` rule of SPM resources. This is done by handling `.process` rules as it was done previously and for `.copy` rules a simple `.folderReference` is used. 

### How to test the changes locally 🧐

- Try to run `tuist fetch`  and `tuist generate` against https://github.com/alexanderwe/tuist-spm-resources-copy-issue and have a look at the generated project 
- The generated project should build fine without duplicated file errors

### Checklist ✅
